src/pages/User/User.tsx

```diff
@@ -85,7 +86,7 @@ const User = (): JSX.Element => {
                     to={`my-profile/${profile?.id}`}
                     label={profile?.name ?? t('nav.profile')}
                     variant="text"
-                    startIcon={<img className={styles.profileIcon} src={profile?.avatar_url} alt={profile?.name} />}
+                    startIcon={<img className={styles.profileIcon} src={profile?.avatar_url || defaultAvatar} alt={profile?.name} />}
```

Comment:
It looks like similar fallback logic is used in several places. Would it make sense to have the profile returned from the store hooks above be a dummy one with the default image by default?

Reply:
https://github.com/jwplayer/ott-web-app/commit/df8266634ac7eaba18352129b93fdf5d38761a25#diff-6c741f0c03d4c3fce6df09209928c5456ed6c41be862f1042159c4ecca8b5efc

Wdyt of this approach?